app/_locales/en/messages.json

```diff
@@ -1088,6 +1088,9 @@
   "gasLimitInfoTooltipContent": {
     "message": "Gas limit is the maximum amount of units of gas you are willing to spend."
   },
+  "gasLimitRecommended": {
+    "message": "Recommended gas limit is $1. If the gas limit is less than estimated transaction can fail."
```

Comment:
I would modify this to " "Recommended gas limit is $1. If the gas limit is less that, it may fail.""

Reply:
@danjm I modified the message as you suggested.
You can see changes here: 8e8a6ecf4a162029c09ca4616d393bde27dc713a
Is it okay now?